### DISCUSSION TITLE
Unavailable state change

### DISCUSSION THREAD

**Discussion by AndyDann** [make-all/tuya-local#647] [General]
Hi,

I have a couple of bulbs that get switched off at the power plug and thus become "unavailable". When power is applied to the lamps the state is not changing to "off" or "on". If I restart the integration or HA then the correct state is returned and all is good.

Is there any to work around to handle the change from "unavailable"???

Cheers

**Comment by GaidinbgAstrabit** [comment]
Hello,

I'm experiencing the same issue - since the connection of my Tuya WiFi smart thermostat  to the local network keeps dropping from time to time the state of the device in the Tuya Local integration first becomes unavailable and then changes to unknown and stays in this state as long as eihter the HA or the integration are reloaded. The "standard" integration with the Tuya Cloud also gets  the state "unavailable" but manages to get the state back when the device reconnects to the network.
